app/assets/stylesheets/govuk_publishing_components/components/_govspeak.scss

```diff
@@ -28,3 +28,12 @@
     text-align: start;
   }
 }
+
+.gem-c-govspeak--inverse {
+  color: govuk-colour('white');
+
+  a {
+    @include govuk-link-common;
+    @include govuk-link-style-inverse;
+  }
+}
```

Comment:
Would this be better placed in [_typography](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss)?

Reply:
Fair point, have moved. I've not nested it in the existing structure of this file as I'm concerned about the volume of outputted CSS, but added it in at the end as a standalone item.